baseline_testing/config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "save_kpi_results": true,
+    "save_measurements": true,
+    "run_user_defined_test": false,
+	
+    "bestest_air": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day", "peak_cool_day", "typical_cool_day", "mix_day"]
+    },
+	
+	"bestest_hydronic_heat_pump": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day"]
+    },
+		
+	"bestest_hydronic_heat_pump": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day"]
+    },
+		
+	"multizone_office_simple_air": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day", "peak_cool_day", "typical_cool_day", "mix_day"]
+    },
+		
+	"multizone_residential_hydronic": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day"]
+    },
+		
+	"singlezone_commercial_hydronic": {
+        "electricity_price": ["constant", "dynamic", "highly_dynamic"],
+        "time_period": ["peak_heat_day", "typical_heat_day"]
+    },
+			
+	"twozone_apartment_hydronic": {
```

Comment:
Is there a reason "highly_dynamic" and "typical_heat_day" are not included as scenarios?

Reply:
Sorry for this unintentional error. This was fixed in the new PR.